src/js/components/container/ButtonContainer.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from "react";
+import ButtonInput from "../presentational/ButtonInput";
+import ButtonStyle from "../presentational/ButtonStyle";
+
+
+class ButtonContainer extends Component {
+  constructor() {
+    super();
+    this.state = {
+      buttonPressed: 0
+    };
+    this.click = this.click.bind(this);
+  }
+  click(event){
```

Comment:
can you change the name to `onClickHandler`

Reply:
Done!